### DISCUSSION TITLE
BicycleNetworkFactor as optional costing option

### DISCUSSION THREAD

**Discussion by caspg** [valhalla/valhalla#4700] [Ideas]
I'm working on a website for cyclists and I'm planing to use Valhalla. I'm focusing mostly on bicycle tourers which are mostly following bicycle route networks. 

BicycleNetworkFactor as optional costing option would be very helpful. Setting `BicycleNetworkFactor` to something like 0.1 would strongly favour networks, right?

If that sounds like a good idea, I could try to prepare a PR.

**Comment by chrstnbwnkl** [comment]
This has been addressed in #3674, but still no PR that addresses the issue. Feel free to give it a shot!
